doc-content/enterprise-only/openshift/authoring-params.adoc

```diff
@@ -27,6 +27,9 @@ Alternatively, if you want to use the Maven repository that is built into {CENTR
 ** *Password for the Maven service hosted by {CENTRAL}* (`{CENTRAL_CAPITAL_UNDER}_MAVEN_PASSWORD`): The password for the built-in Maven repository.
 endif::DM[]
 +
+. You can use Git hooks to facilitate interaction between the internal Git repository of {CENTRAL} and an external Git repository. To configure Git hooks, set the following parameter:
+** *Git hooks directory* (`GIT_HOOKS_DIR`): The fully qualified name of a Git hooks directory, for example, `/opt/eap/standalone/data/kie/git/hooks`. You must provide this directory and mount it at this path; for instructions, see <<githooks-proc>>.
```

Comment:
I would put Git hooks directory after GIT_HOOKS_DIR, as the parameters are listed in the next step. 
Consider: You must specify this directory path and mount it at the specified path.